scripts/Get-TagsDocumentation.ps1

```diff
@@ -1,30 +1,16 @@
 param(
     [string]$Branch='master',
-    [string]$ImageBuilderImageName='microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20171122115946',
-    [string]$RepoName
```

Comment:
Are this file's changes just cleanup because of the repo merge(s)?

Reply:
No - I needed to update the ImageBuilder in order to pickup the [changes ](https://github.com/dotnet/docker-tools/pull/90) to generate the appropriate 1709 header.  While I was touching this file, I decided to cleanup the unnecessary logic to retrieve the RepoName.  This logic is a result of coping the script from https://github.com/dotnet/dotnet-docker.  There is no need for this - Let's follow the KISS principle.